client/js/test/table-view.test.js

```diff
@@ -8,34 +8,109 @@ describe('table-view', () => {
         const fixturePath = './client/js/test/fixtures/sheet-container.html';
         const html = fs.readFileSync(fixturePath, 'utf8');
         document.documentElement.innerHTML = html;
-    })
+    });
+
+    describe('highlighting entire columns and rows', () => {
```

Comment:
Added edge cases that test for non-numeric values inputted in the table data, and how they would be effectively ignored by the summation.